src/jobsautoreport/query.py

```diff
@@ -29,18 +30,25 @@ def _get_query_all_jobs(from_date: datetime, to_date: datetime) -> dict:
                                 }
                             }
                         }
-                    ],
+                    ]
                 }
             }
         }
 
     @staticmethod
-    def _get_query_successful_jobs(from_date: datetime, to_date: datetime) -> dict:
+    def _get_query_e2e_jobs_by_type(
+        from_date: datetime, to_date: datetime, job_type: JobType
+    ) -> dict:
         return {
             "query": {
                 "bool": {
                     "must": [
-                        {"match": {"job.state": "success"}},
+                        {"match": {"job.name": {"query": "e2e", "operator": "and"}}},
```

Comment:
for non-rehearsal jobs, we might want to filter on `assisted-service`, `assisted-installer`, `assisted-installer-agent` and `assisted-image-service` repos in `openshift` organization.